src/checks/components/CheckCard.tsx

```diff
@@ -37,6 +37,7 @@ import {Check, Label} from 'src/types'
 
 // Utilities
 import {relativeTimestampFormatter} from 'src/shared/utils/relativeTimestampFormatter'
+import ErrorBoundary from '../../shared/components/ErrorBoundary'
```

Comment:
```suggestion
import ErrorBoundary from 'src/shared/components/ErrorBoundary'
```

Reply:
Did a global search for `import ErrorBoundary from '../ `and cleaned those all up in my latest commit.